app/frontend/src/components/farmopscreening/Step1.vue

```diff
@@ -191,9 +188,9 @@
         </v-row>
       </v-container>
 
-      <BaseHeaderSub :text="'COVID-19 Coordinator'" />
+      <BaseHeaderSub v-if="formVersionId == 1" :text="'COVID-19 Coordinator'" />
```

Comment:
Make sure to prefer using hard-casted === checks whenever possible as it's safer.